### PR TITLE
WIP: Reintroduce FOSUserBundle

### PR DESCRIPTION
**Pull request by Nyholm**
| Q             | A
| ------------- | ---
| License       | MIT

Initially added in #270 and reverted in #655

### CODE REVIEW

**Comment by maxhelias** [comment]
Blocked by FriendsOfSymfony/FOSUserBundle#2708

File: friendsofsymfony/user-bundle/2.1/config/packages/fos_user.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+fos_user:
+    db_driver: no_driver # valid values are 'orm', 'mongodb' and 'couchdb'
```
Comment (Nyholm) [inline friendsofsymfony/user-bundle/2.1/config/packages/fos_user.yaml]: This is not yet supported.

**Comment by Nyholm** [comment]
We are waiting for https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2708

**Comment by juanmiguelbesada** [comment]
I think we have a problem here.

One side, we don't want to pre-config the db_driver to orm.

But at the other side, we are includin a User entity that uses ORM annotations, that requires doctrine/orm.

I think we have two options right now.

One, set default db_driver to orm, and configure a the recipe according.

The other, create a new meta-package, fosuserbundle-orm, that requires fosuserbundle and doctrine-bundle, and create a new recipe for this new meta-package that can be configured to use orm as db_driver.

What do you think?

**Pull request closed without merging**